less/switchControls.less

```diff
@@ -36,6 +36,19 @@
       }
     }
   }
+  
+  &.small {
```

Comment:
This change affects every switch controls outside of `about:preferences` too. did you confirm this does not regress something?